quickstarts/microsoft.machinelearningservices/machine-learning-job-create-or-update-job/modules/codeversion.bicep

```diff
@@ -0,0 +1,77 @@
+@description('Specifies the name of the Azure Machine Learning workspace where sweep job will be deployed')
+param workspaceName string
+
+@description('Name of the blob as it is stored in the blob container')
+param filename string = 'hello_world.py'
+
+@description('Name of the blob container')
+param containerName string = 'workspaceblobstore'
+
+@description('Azure region where resources should be deployed')
+param location string = resourceGroup().location
+
+@description('Desired name of the storage account')
+param storageAccountName string
+
+@description('Specifies the env version for sweep job.')
+param codeVersion string = '1'
+
+@description('Specifies the env for command job.')
+param codeId string = 'code'
+
+// fetching existing storage account
+resource storage 'Microsoft.Storage/storageAccounts@2021-04-01' existing = {
```

Comment:
remove this and the deployment script and use a staged artifact as per repo best practices, see:

See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle artifacts

Reply:
I was able to remove the deployment script, but if I attempt to remove the storage here, I will not be able to successfully create since I need the storage and blobService for naming the codeversion

Reply:
ok, I'm still seeing the script (possible you haven't pushed those changes yet)